modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -31,22 +31,40 @@ const onSetupToggle = (editor: Editor, formatName: string, formatValue: string)
   };
 };
 
+const onSetupToggleSingle = (editor: Editor, formatName: string, formatValue: string) => {
+  return onSetupToggle(editor, formatName, () => editor.formatter.match(formatName, { value: formatValue }));
+};
```

Comment:
```suggestion
const onSetupToggleSingle = (editor: Editor, formatName: string, formatValue: string) =>
  onSetupToggle(editor, formatName, () => editor.formatter.match(formatName, { value: formatValue }));
```
Shorthand functions please